app/services/data_stage/process_school_changes.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+module DataStage
+  class ProcessSchoolChanges < ::BaseService
+    include GiasTypes
+
+    UNHANDLED_ATTRIBUTES = %w[
+      school_type_code
+      school_type_name
+      section_41_approved
+      ukprn
+    ].freeze
+
+    def call
+      DataStage::SchoolChange.includes(:school).unhandled.status_changed.find_each do |change|
+        next if has_unhandled_changes?(change)
```

Comment:
Does the word "unhandled" on these two lines mean different things?

Reply:
Yes, bit confusing. `SchoolChange` has a handled flag on the record to indicate whether it's been dealt with.  The `has_unhandled_changes?` means it contains changes to attributes that we don't want/know how to deal with yet

Reply:
Maybe we should rename one of the 2?

Reply:
I'll rename the scope to `unprocessed`